packages/terra-open-source-scripts/src/terra-cli/prepare-for-release/prepareForReleaseMonoRepoHandler.js

```diff
@@ -20,13 +20,24 @@ module.exports = async () => {
     // to capture the output to a file
     childProcess.execSync(`script -q ${VERSION_OUTPUT_PATH} lerna version --no-git-tag-version`, { stdio: 'inherit' });
 
-    if (stripAnsi(await fs.readFile(VERSION_OUTPUT_PATH, 'utf-8')).includes('lerna success version finished')) {
-      const { stdout } = await spawn('npx', ['lerna', 'changed'], { stdioString: true });
-      // The lines we're looking for don't start with lerna. They're the ones with the changed packages
-      const updatedPackages = stdout.split('\n').filter(x => !x.startsWith('lerna')).map(x => `packages/${x.replace('@cerner/', '')}`);
+    const versionOutputContents = stripAnsi(await fs.readFile(VERSION_OUTPUT_PATH, 'utf-8'));
+    if (versionOutputContents.includes('lerna success version finished')) {
+      // Extract packages and versions from the version output
+      const packagesAndVersionsRegEx = /- (\S+): \S+ => (\d+.\d+.\d+)/g;
+      let packagesAndVersionsMatch = packagesAndVersionsRegEx.exec(versionOutputContents);
+      const packagesAndVersions = [];
+      const updatedPackages = [];
+      while (packagesAndVersionsMatch !== null) {
+        updatedPackages.push(packagesAndVersionsMatch[1]);
+        packagesAndVersions.push(`${packagesAndVersionsMatch[1]}@${packagesAndVersionsMatch[2]}`);
+        packagesAndVersionsMatch = packagesAndVersionsRegEx.exec(versionOutputContents);
+      }
 
       // Update release version in changelog files
-      await Promise.all(updatedPackages.map(packagePath => updateChangelogForPackage(packagePath)));
+      const packagePaths = updatedPackages.map(x => `packages/${x.replace('@cerner/', '')}`);
```

Comment:
why are you removing `@cerner`? i think it makes sense to keep it in these cases

Reply:
actually maybe i'm confused. I think it makes sense to keep `@cerner` in the console message, but here you're doing something else.